sdk/storage/azure-storage-file-share/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --v3 --python
 
 ### Settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.FileStorage/preview/2023-01-03/file.json
+input-file: https://raw.githubusercontent.com/tasherif-msft/azure-rest-api-specs/6aa9b7675e9f306a077a60f3d9405e7fc5e809e1/specification/storage/data-plane/Microsoft.FileStorage/preview/2024-02-04/file.json
```

Comment:
Has this Swagger change merged to main yet? If not, we may want to merge this to a feature branch instead so we don't have Tamer's fork going into `main`.

Reply:
Good catch, this is actually supposed to target the feature branch so I just swapped it over!